pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -257,8 +259,8 @@ def work(self):
             return WorkerResult.SUCCESS
 
         if self.config['snipe']:
-            if self.config['snipe_high_prio_only']:
-                if self.config['snipe_high_prio_threshold'] < pokemon['priority'] or pokemon['is_vip']:
+            if self.config.get('snipe_high_prio_only', False):
+                if self.config.get('snipe_high_prio_threshold', 400) < pokemon['priority'] or pokemon['is_vip']:
```

Comment:
This all looks great except these. These defaults should go up after line 87 with the other config defaults for this file.